front/components/assistant/conversation/Conversation.tsx

```diff
@@ -161,7 +161,7 @@ export default function Conversation({
   }
 
   return (
-    <div className="pb-32">
+    <div className="pb-72">
```

Comment:
This is pb-72 because the input bar has a m-h-64. pb-64 is not enough.

Reply:
That means we have a big blank when the input bar is not big. How we sure we want that? 🙏🏻

Reply:
Did you check how it impacts auto-scrolling for long conversations?

There's also a world in which we don't fix this. On this one the boundary between bug and feature is quite blurry